docs/source/modules/connections.rst

```diff
@@ -7,4 +7,5 @@ Connection Object
    :members:
    :exclude-members: DataError, DatabaseError, Error, InterfaceError,
                      IntegrityError, InternalError, NotSupportedError,
-                     OperationalError, ProgrammingError, Warning
+		     OperationalError, ProgrammingError, Warning,
+		     escape, literal, write_packet
```

Comment:
Hmm that's weird. I use autopep8 to avoid these problems, but maybe I had turned it off for some reason. Will fix when I get to the office sorry.